.github/workflows/deploy-ipfs.yml

```diff
@@ -0,0 +1,112 @@
+name: Publish to IPFS
+
+on:
+  # push:
+  #   branches:
+  #     - master
+  #     - develop
+  pull_request:
+    branches:
+      # - master
+      - develop
+
+env:
+  NODE_VERSION: '14.x'
+  IPFS_GATEWAY: 'www.cloudflare-ipfs.com.'
+  # neeeds to be set, as build currently fails because of warnings
+  CI: false
+  AWS_HOSTED_ZONE_ID: 'Z02908602B6KLAQBRUBST'
+  AWS_DEFAULT_REGION: 'us-east-1'
+  DOMAIN_NAME: "${{ endsWith(github.ref, '/master') && 'yearn.finance' || 'b3tools.xyz' }}"
+  REACT_APP_ETHERSCAN_API_KEY: 'GEQXZDY67RZ4QHNU1A57QVPNDV3RP1RYH4'
+  REACT_APP_ALCHEMY_API_KEY: 'euSwyu6Yf-VQ3NJ32KHxDhHmTta7OvIe'
```

Comment:
keys being public?

Reply:
These were taken from .env.example, so pretty much public already. Just have to integrate environments into the pipeline as well, and then could have different secrets for each env.

Reply:
Couldnt we use the variables from .env file directly? Do we need to have this here on public file duplicated?